tests/debugger/do_throw.js

```diff
@@ -0,0 +1,19 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+function inevitable() {
+  print("An error will be thrown.");
+}
+
+inevitable();
```

Comment:
Just curious: why is any code needed in this file for this test case?

Reply:
Well, we could leave it empty as well.

Reply:
I don't think it hurts to keep the current file.

Reply:
I was just wondering... I thought perhaps it was needed to get a breakpoint opcode emitted or something.